src/views/MidiView.vue

```diff
@@ -75,6 +104,18 @@ onMounted(async () => {
               {{ (input.manufacturer || "(Generic)") + ": " + input.name }}
             </option>
           </select>
+          <div class="control">
```

Comment:
I have no idea what I'm doing with this CSS. I would like to have the labels under the check boxes like we have in SW1.

Reply:
There is a div element which wraps around the 16 span elements for the input channels checkboxes. If you give that an id such as input-channels-wrapper or something, then you can add this css:

```
#input-channels-wrapper {
    column-gap: 1rem;
}
#input-channels-wrapper span {
    display: flex;
    flex-flow: column;
    text-align: center;
}
```

I think that will make the check boxes appear the way you want.

![Screenshot_20220702_142452](https://user-images.githubusercontent.com/11957541/177002800-9c6a4f7c-7bfc-46d6-87bf-4e516b577df8.png)